src/multiversioning.zig

```diff
@@ -1402,37 +1412,48 @@ pub fn self_exe_path(allocator: std.mem.Allocator) ![:0]const u8 {
 
     const native_self_exe_path = try std.fs.selfExePath(&buf);
 
-    if (builtin.target.os.tag == .linux and std.mem.eql(
-        u8,
-        native_self_exe_path,
-        "/memfd:" ++ multiversion_uuid ++ " (deleted)",
-    )) {
-        // Technically, "/memfd:tigerbeetle-multiversion-... (deleted)" is a valid path at which you
-        // could place your binary - please don't!
+    // Technically, "/memfd:tigerbeetle-multiversion-... (deleted)" is a valid path at which you
```

Comment:
Oh, this comment was directly above the statFile, since that's what it's checking - we're trying to stat the path that really shouldn't exist to make sure it doesn't.